source/_docs/tmp.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Temporary File Management
+description: Understand Pantheon's default temporary path and learn how to debug .tmp file errors.
+tags: [debugcode, infrastructure]
+categories: []
+---
+## Default Temporary Path  
+Pantheon configures an appropriate temporary path for [WordPress](https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L83-L86){.external} and [Drupal 8](https://github.com/pantheon-systems/drops-8/blob/master/sites/default/settings.pantheon.php#L146-L154){.external}. Drupal 7 sites can achieve the same configuration by adding the following to `settings.php`:
```

Comment:
The contents of GitHub links on the `master` branch may change as more commits are added. To avoid this, link to a specific tag:

- [WordPress](https://github.com/pantheon-systems/WordPress/blob/4.9.6/wp-config.php#L84:L86)
- [Drupal 8](https://github.com/pantheon-systems/drops-8/blob/8.5.3/sites/default/settings.pantheon.php#L152:L154)